Makefile

```diff
@@ -151,7 +151,7 @@ test-unstable:
 
 .PHONY: test-fullstack
 test-fullstack:
-	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=9 PROVE_ARGS="$$HARNESS t/full-stack.t"
+	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=9 PROVE_ARGS="$$HARNESS t/full-stack.t" RETRY=3
```

Comment:
Maybe #3504 could help here - although if you just want it out of the way for now, I'm okay with that

Reply:
yes, that's what I want :)

Reply:
#3562